### PR TITLE
feat: add multiple theme support

### PR DESCRIPTION
**Pull request by Rickk137**
https://sovryn.monday.com/boards/2218344956/views/63221561/pulses/3015316510

### CODE REVIEW

File: apps/frontend/src/app/5_pages/App.tsx [hunk 1]
```diff
@@ -2,7 +2,18 @@ import React from 'react';
 import { Button, Lead } from '@sovryn/ui';
 import styles from './index.module.css';
 
+enum Theme {
```
Comment (tiltom) [inline apps/frontend/src/app/5_pages/App.tsx]: Rename it to `AppTheme`, create a new file (e.g. `tailwind.ts`) in `types` folder and move it there.

**Review by tiltom — CHANGES_REQUESTED** [review]

File: apps/frontend/src/app/5_pages/App.tsx [hunk 2]
```diff
@@ -1,8 +1,11 @@
...
-import styles from './index.module.css';
+import React from "react";
+import { Button, Lead } from "@sovryn/ui";
+import styles from "./index.module.css";
+import { AppTheme } from "../../types/tailwind";
+import { useTheme } from "../../hooks/useTheme";
 
 function App() {
+  const { handleThemeChange } = useTheme();
   return (
```
Comment (tiltom) [inline apps/frontend/src/app/5_pages/App.tsx]:
```suggestion
  const { handleThemeChange } = useTheme();
  
  return (
```

File: apps/frontend/src/hooks/useTheme.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { useEffect } from "react";
+import { AppTheme } from "../types/tailwind";
+
+export const useTheme = () => {
+  const handleThemeChange = (theme: AppTheme) => {
```
Comment (tiltom) [inline apps/frontend/src/hooks/useTheme.ts]: Use `useCallback`.

File: apps/frontend/src/hooks/useTheme.ts [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    document.documentElement.setAttribute("data-theme", theme);
+    localStorage.setItem("theme", theme);
+  };
+
+  useEffect(() => {
+    const theme = localStorage.getItem("theme");
+    console.log(theme)
```
Comment (tiltom) [inline apps/frontend/src/hooks/useTheme.ts]:
```suggestion

```

**Review by tiltom — CHANGES_REQUESTED** [review]

**Review by tiltom — APPROVED** [review]

**Comment by tiltom** [comment]
It looks good but deployment is failing, pls fix it @Rickk137 .

**Review by tiltom — APPROVED** [review]

**Review by pietro-maximoff — APPROVED** [review]

**Pull request merged**